packages/apollo-gateway/src/FieldSet.ts

```diff
@@ -10,6 +10,7 @@ import {
   GraphQLObjectType,
 } from 'graphql';
 import { getResponseName } from './utilities/graphql';
+import _ from 'lodash';
```

Comment:
This is just a nitpick, but I'd prefer not to introduce `lodash` if we don't need to. In this case:
* `partition` exists within this repo under `/utilities/array.ts`.
* `groupBy` already exists within this repo, it could be moved from `FieldSet.ts` to the `/utilities/array.ts` file if the implementation is sufficient for your use case.

Reply:
Sidenote, if for some reason we land on keeping it as-is based on discussion here, `lodash` needs to be added as a dependency to the gateway repo since it currently is not.